test/indexer/test_indexer.py

```diff
@@ -2113,6 +2114,54 @@ def test_disallow_manifest_column_joiner(self):
         with self.assertRaisesRegex(RequirementError, "'||' is disallowed"):
             self._index_bundle(bundle)
 
+    def test_bionetwork_fields(self):
+        bundle_fqid = self.bundle_fqid(uuid='3ac62c33-93e1-56b4-b857-59497f5d942d',
```

Comment:
Why did you pick this can?

Reply:
I wanted one that would index quickly, so I picked the one with the smallest file size.